rules-java-archives/addon/pom.xml

```diff
@@ -7,12 +7,15 @@
         <groupId>org.jboss.windup.rules.apps</groupId>
         <artifactId>rules-java-archives-parent</artifactId>
         <version>2.0.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
```

Comment:
Oh, nvm... I didn't realize that was the default. Thanks for pointing that out. :)